.golangci.next.reference.yml

```diff
@@ -2106,6 +2108,45 @@ linters:
         - '^math\.'
         - '^http\.StatusText$'
 
+    modernize:
```

Comment:
Great to finally see this in here! 🚀 What are the default values for this linter?